src/components/like-button/like-button.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
+import { Toggle } from '../toggle/toggle';
+import { HeartIcon } from '../icons';
+
+interface LikeButtonProps {
+	count: number;
+	onClick: () => void;
+}
+
+const getLikesText = (count: number) => (count > 1 ? `${count} Likes` : `${count ? count + ' ' : ''}Like`);
+
+const LikeButton: React.FC<LikeButtonProps> = ({ count, onClick }) => {
```

Comment:
The index.ts file is missing for the like button

Reply:
@malinovic fixed